plugins/must-gather/skills/must-gather-analyzer/scripts/analyze_ovn_dbs.py

```
@@ -0,0 +1,444 @@
+    tarball = network_logs / "ovnk_database_store.tar.gz"
+    extract_dir = network_logs / "ovnk_database_store"
+
+    if not tarball.exists():
+        print(f"Error: Database tarball not found: {tarball}", file=sys.stderr)
+        return None
+
+    # Extract if directory doesn't exist
+    if not extract_dir.exists():
+        print(f"Extracting {tarball}...")
+        with tarfile.open(tarball, 'r:gz') as tar:
+            tar.extractall(path=network_logs)
```

Ruff (0.14.3)

Uses of `tarfile.extractall()`

(S202)